vocabularies/UI.xml

```diff
@@ -1073,6 +1117,19 @@ The trend is
         <Property Name="RequestAtLeast" Type="Collection(Edm.PropertyPath)" Nullable="false">
           <Annotation Term="Core.Description" String="Properties that should always be included in the result of the queried collection" />
         </Property>
+        <Property Name="RequestAtLeastDynamic" Type="Collection(Edm.AnnotationPath)" Nullable="false">
+          <Annotation Term="Core.Description" String="Dynamic properties introduced by annotations that should always be included in the result of the queried collection" />
```

Comment:
How would that work? Could you give a request example?

Reply:
If the dynamic property comes from an aggregation, it shall be requested through an `$apply=aggregate(...)` clause.

Reply:
Where do we describe the client logic necessary to construct such a request?

Reply:
To be discussed in Albatross first.

Reply:
Albatross: 
- At the moment no use case for `RequestAtLeastDynamic` is known. Hence skipping it for the moment.
- The explanation for `RequestAtLeast` that it translates into `$select` (no `$apply`) or `groupby` (in `$apply`) is missing.